pulp_rpm/app/models/repository.py

```diff
@@ -220,10 +220,10 @@ class RpmRepository(Repository):
 
     autopublish = models.BooleanField(default=False)
     metadata_checksum_type = models.CharField(
-        default=CHECKSUM_TYPES.SHA256, choices=CHECKSUM_CHOICES, max_length=10
+        choices=CHECKSUM_CHOICES, max_length=10
     )
     package_checksum_type = models.CharField(
-        default=CHECKSUM_TYPES.SHA256, choices=CHECKSUM_CHOICES, max_length=10
+        choices=CHECKSUM_CHOICES, max_length=10
```

Comment:
The solution looks good to me overall.
I'm not sure if removing the default value from the model makes this field mandatory at the DB level.
Does empty CharField defult to an empty string? Do we need to explicitly allow blank or null value?

Reply:
Yes, I'll make it nullable